MapboxGeocoder/MBGeocoder.swift

```diff
@@ -166,6 +175,7 @@ open class Geocoder: NSObject {
      */
     @objc public convenience init(accessToken: String?) {
         self.init(accessToken: accessToken, host: nil)
+        self.initSession()
```

Comment:
An instance of this object is potentially created when the application launches and lasts for the application’s lifetime. The application is also free to create an arbitrary number of these objects at any time, without making any statement about the relationship between the Geocoder instance and a query.